src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1461,21 +1462,39 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
         $container->setAlias('messenger.transport.encoder', $config['encoder']);
         $container->setAlias('messenger.transport.decoder', $config['decoder']);
 
+        if (null === $config['default_bus']) {
+            if (count($config['buses']) > 1) {
+                throw new LogicException(sprintf('You need to define a default bus with the "default_bus" configuration. Possible values: %s', implode(', ', array_keys($config['buses']))));
+            }
+
+            $config['default_bus'] = array_keys($config['buses'])[0];
+        }
+
+        $defaultMiddlewares = array('before' => array('logging'), 'after' => array('route_messages', 'call_message_handler'));
```

Comment:
Isn't `validation` supposed to be in `defaultMiddlewares['before']` if the component is enabled, as described in https://github.com/symfony/symfony/pull/26864#issuecomment-383301982 ?

Reply:
See https://github.com/symfony/symfony/pull/26864#discussion_r183215246

Reply:
Alright, let's stick with it :)